sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/ResourceTests/GeneratedResourcesTest.cs

```diff
@@ -37,7 +37,7 @@ namespace SonarAnalyzer.UnitTest.ResourceTests
     [TestClass]
     public class GeneratedResourcesTest
     {
-        private const string RspecRelativeFolderPath = @"..\..\..\..\rspec\";
+        private const string RspecRelativeFolderPath = @"..\..\..\..\..\rspec\";
```

Comment:
the framework is now present in the bin path so we need to get up one more folder.